docs/actions/live-events.mdx

```diff
@@ -10,13 +10,52 @@ You can stream events (sign-ups, logins, machine-to-machine tokens issued, and m
 your Ory Network project, to your own infrastructure. Pipe those events into your own data warehouse, data lake, or flavor of
 choice, and use them to power your own analytics, dashboards, data science, and more.
 
-Live event streams are available for Ory Network enterprise contracts. Talk to your account manager or reach out
-[directly](https://www.ory.sh/contact/) to find out more.
+Live event streams are available for Ory Network enterprise contracts. Talk to your account manager or
+[reach out directly](https://www.ory.sh/contact/).
 
-## Event descriptions
+## Event shape
 
-Ory emits events for many different actions. The following is a list of all events that are currently supported. This list is not
-exhaustive, and there might by additional events being emitted.
+Ory emits events for many different actions. All events abide by the same schema:
+
+```json
+{
+  "name": string,       // The unique name of the event (e.g., "IdentityCreated").
+  "version": string,     // The numerical schema version of the event as a string.
+  "timestamp": string,  // ISO 8601 formatted timestamp when the event occurred.
+  "projectId": string,  // The ID of the Ory project the event belongs to.
+  "eventAttributes": map<string, string> // Additional event-specific attributes.
+}
+```
+
+Example:
+
+```json
+{
+  "name": "WebhookSucceeded",
+  "version": "1",
+  "timestamp": "2025-05-16 11:18:56",
+  "eventAttributes": {
+    "GeoLocationCity": "Munich",
+    "WebhookID": "",
+    "WebhookTriggerID": "18093e08-0f72-445f-aacb-151fc777a232"
+  }
+}
+```
+
+**Important:**
```

Comment:
```suggestion
```

please avoid bolding in general except if you talk about UI elements. 
if you want to put extra emphasis on this section you can put it in an h4 (preferred) or an admonition (e.g. :::caution - but use this sparingly please)